src/avs/task/TaskAVSRegistrarBase.sol

```diff
@@ -93,4 +97,22 @@ abstract contract TaskAVSRegistrarBase is
         avsConfig = config;
         emit AvsConfigSet(config.aggregatorOperatorSetId, config.executorOperatorSetIds);
     }
+
+    /// @notice Before registering operator, check if the operator is in the allowlist
+    /// @dev Reverts for:
+    ///      - OperatorNotInAllowlist: The operator is not in the allowlist
```

Comment:
Small nit about the natspec following the same format as the other internal functions.